librz/core/panels.c

```diff
@@ -1720,30 +1715,6 @@ bool __handle_window_mode(RzCore *core, const int key) {
 		free (cmd);
 	}
 		break;
-	case 'h':
-		(void)__move_to_direction (core, LEFT);
```

Comment:
do you really need to remove this? it doesn't seem to be used only when snow/fun is used, is it?

Reply:
It is only used in snow

Reply:
oh wait you're right

Reply:
woups, fixed